packages/angular/cli/utilities/tty.ts

```diff
@@ -11,5 +11,5 @@ export function isTTY(): boolean {
     return !(force === '0' || force.toUpperCase() === 'FALSE');
   }
 
-  return !!process.stdout.isTTY && !!process.stdin.isTTY;
+  return !!process.stdout.isTTY && !!process.stdin.isTTY && !('CI' in process.env);
```

Comment:
I wonder how common it is for people to set `CI=false`/`CI=0` on their non-CI envs 😆